SimG4CMS/HGCalTestBeam/plugins/HGCPassive.cc

```diff
@@ -4,12 +4,13 @@
 // Description: Main analysis class for HGCal Validation of G4 Hits
 ///////////////////////////////////////////////////////////////////////////////
 
-#include "HGCPassive.h"
+#include "SimG4CMS/HGCalTestBeam/interface/HGCPassive.h"
```

Comment:
@bsunanda , I am sorry, but recent requirement from @smuzaffar was to have include files for plugin inside plugin directory. This include files should not be used by any other class except producer itself HGCPassive.cc, normally these two files should be merged together into HGCPassive.cc, two separate files only if the producer is too big.